backends/dpdk/dpdkArch.cpp

```diff
@@ -2957,6 +2957,14 @@ MoveNonHeaderFieldsToPseudoHeader::addAssignmentStmt(const IR::Expression *e) {
 const IR::Node *MoveNonHeaderFieldsToPseudoHeader::postorder(IR::AssignmentStatement *assn) {
     if (is_all_args_header) return assn;
     auto result = new IR::IndexedVector<IR::StatOrDecl>();
+    if (isLargeFieldOperand(assn->left) && assn->right->is<IR::Constant>()) {
+        auto leftType = assn->left->type->to<IR::Type_Bits>()->width_bits();
+        auto rightType = assn->right->type->to<IR::Type_Bits>()->width_bits();
+        if (leftType == 128 && rightType == leftType) {
```

Comment:
128 should be static constant that you define, maybe in `dpdkHelpers.h`. It should be used everywhere you use "128".

Reply:
Updated

Reply:
I would consider wrapping this in a namespace or struct to avoid conflicts. You can then access the value like `DpdkConstants::SupportedInstructionWidth` or something similar. Capturing what this restriction is meant to represent.